index.js

```diff
@@ -14,6 +14,7 @@ function Plugin(options) {
   if (this.options.logTime === undefined) {
     this.options.logTime = DEFAULT_LOG_TIME;
   }
+  this.options.append = options.append || false;
```

Comment:
Actually, we could accept this patch if this is hidden behind a flag which is off by default. At least until we are sure append is what we want as default.